widget/entry.go

```diff
@@ -309,6 +313,12 @@ func (e *Entry) Enable() { // TODO remove this override after ReadOnly is remove
 	e.DisableableWidget.Enable()
 }
 
+// SetOnChanged sets the onChanged and returns the entry to allow chaining
```

Comment:
Is this really part of the Data Binding API?

Reply:
no.

I think I ended up attending to numerous deprecation warnings outside the scope of this.  Should have done these updates in a different PR